### PR TITLE
basic-features.md: add the column for v8.0.0

### PR DESCRIPTION
**Pull request by hfxsd**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/16796
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by Frank945946 — APPROVED** [review]

**Comment by dveeden** [comment]
@hfxsd maybe we can remove the v4.0 column to ensure this is readable and renders will on desktop, mobile and PDF?

File: basic-features.md [hunk 1]
```diff
@@ -22,241 +22,241 @@ You can try out TiDB features on [TiDB Playground](https://play.tidbcloud.com/?u
...
+| [Runaway Queries management](/tidb-resource-control.md#manage-queries-that-consume-more-resources-than-expected-runaway-queries) | E | E | E | N | N | N | N | N | N | N | N | N |
+| [Background tasks management](/tidb-resource-control.md#manage-background-tasks) | E | E | E | N | N | N | N | N | N | N | N | N |
+| [TiFlash Disaggregated Storage and Compute Architecture and S3 Support](/tiflash/tiflash-disaggregated-and-s3.md) | Y | Y | Y | E | N | N | N | N | N | N | N | N |
+| [Selecting TiDB nodes for the Distributed eXecution Framework (DXF) tasks](/system-variables.md#tidb_service_scope-new-in-v740) | Y | Y | Y | N | N | N | N | N | N | N | N | N |
+| PD Follower Proxy (controlled by [`tidb_enable_tso_follower_proxy`](/system-variables.md#tidb_enable_tso_follower_proxy-new-in-v530)) | Y | Y | Y | Y | Y | Y | Y | Y | N | N | N | N |
+| [Active PD Follower](/tune-region-performance.md#use-the-active-pd-follower-feature-to-enhance-the-scalability-of-pds-region-information-query-service) (controlled by [`pd_enable_follower_handle_region`](/system-variables.md#pd_enable_follower_handle_region-new-in-v760)) | E | E | N | N | N | N | N | N | N | N | N | N |
```
Comment (qiancai) [inline basic-features.md:259]:
```suggestion
| [Active PD Follower](/tune-region-performance.md#use-the-active-pd-follower-feature-to-enhance-the-scalability-of-pds-region-information-query-service) (controlled by [`pd_enable_follower_handle_region`](/system-variables.md#pd_enable_follower_handle_region-new-in-v760)) | E | E | N | N | N | N | N | N | N | N | N | N |
| [PD 微服务](pd-microservices.md) | Y | N | N | N | N | N | N | N | N | N | N | N |
```

File: basic-features.md [hunk 2]
```diff
@@ -22,241 +22,241 @@ You can try out TiDB features on [TiDB Playground](https://play.tidbcloud.com/?u
...
```
Comment (qiancai) [inline basic-features.md:260]:
```suggestion
| [PD 微服务](pd-microservices.md) | Y | N | N | N | N | N | N | N | N | N | N | N |

```

File: basic-features.md [hunk 3]
```diff
@@ -22,241 +22,241 @@ You can try out TiDB features on [TiDB Playground](https://play.tidbcloud.com/?u
...
+| [DDL algorithm assertions](/sql-statements/sql-statement-alter-table.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| Multi-schema change: add columns | Y | Y | Y | Y | Y | E | E | E | E | E | E | E |
+| [Change column type](/sql-statements/sql-statement-modify-column.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | N | N |
+| [Temporary tables](/temporary-tables.md) | Y | Y | Y | Y | Y | Y | Y | Y | N | N | N | N |
+| Concurrent DDL statements | Y | Y | Y | Y | Y | N | N | N | N | N | N | N |
+| [Acceleration of `ADD INDEX` and `CREATE INDEX`](/system-variables.md#tidb_ddl_enable_fast_reorg-new-in-v630) | Y | Y | Y | Y | Y | N | N | N | N | N | N | N |
+| [Metadata lock](/metadata-lock.md) | Y | Y | Y | Y | Y | N | N | N | N | N | N | N |
+| [`FLASHBACK CLUSTER`](/sql-statements/sql-statement-flashback-cluster.md) | Y | Y | Y | Y | Y | N | N | N | N | N | N | N |
+| [Pause](/sql-statements/sql-statement-admin-pause-ddl.md)/[Resume](/sql-statements/sql-statement-admin-resume-ddl.md) DDL | Y | Y | Y | N | N | N | N | N | N | N | N | N |
+| [TiDB DDL V2](/ddl-v2.md) | E | E | N | N | N | N | N | N | N | N | N | N |
```
[the remark this answers is not in the document]
Comment (hfxsd) [reply basic-features.md:135]: @Frank945946 PTAL

[the remark this answers is not in the document]
Comment (Frank945946) [reply basic-features.md:135]: LGTM

File: basic-features.md (hunk 2 above)
Comment (Frank945946) [reply basic-features.md:260]: LGTM

**Review by Frank945946 — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**